apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/tips.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { select } from '@wordpress/data';
+
+function tipDescription( text, conversion, textFallback ) {
+	if ( typeof createInterpolateElement !== 'undefined' ) {
+		return createInterpolateElement( text, conversion );
+	}
+
+	return textFallback;
+}
+
+const tips = [
+	{
+		context: 'theme',
+		keywords: [ 'theme', __( 'theme' ) ],
+		description: tipDescription(
+			__(
+				'theme - You can visit the <a>theme directory</a> to select a different design for your site.'
```

Comment:
I'm not sure how I feel about prepending the search term. It doesn't seem to be part of the original mockups?

Reply:
Oh, I did again. I though the prepended context was part of the description. Updating.
<img width="426" alt="Screen Shot 2020-05-12 at 7 42 16 AM" src="https://user-images.githubusercontent.com/77539/81674825-32309680-9424-11ea-826c-9832e8151233.png">